### PR TITLE
Flaw References and Comments support

### PR DESCRIPTION
**Pull request by JakubFrejlach**
Added support for Flaw References and Flaw Comments via OSIDB entities.

This PR should be coordinated with [PR which brings additional functionality into OSIDB bindings](https://github.com/RedHatProductSecurity/osidb-bindings/pull/14).

### CODE REVIEW

**Comment by JimFuller-RedHat** [comment]
lgtm ... might be worth including as a separate commit to this PR - bump python cryptographic module (as per github reported vuln)

File: griffon/commands/entities/osidb.py [hunk 1]
```diff
@@ -201,6 +214,324 @@ def create_flaw(ctx, **params):
...
+        exclude=["uuid", "created_dt"],
+    )
+    params = multivalue_params_to_csv(params)
+
+    session = OSIDBService.create_session()
+
+    data = {field: "" for field in fields}
+    data.update((field, value) for field, value in params.items() if value is not None)
+
+    if ctx.obj["EDITOR"]:
```
Comment (JimFuller-RedHat) [inline griffon/commands/entities/osidb.py:428]: does no-editor flag interact with EDITOR ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
-
+### Added
+- CRUD operations for OSIDB entities Flaw Comment (create, retreive, list),
```
Comment (JimFuller-RedHat) [inline CHANGELOG.md:9]: is it intentional not having a delete for Flaw Comment ?

File: griffon/commands/entities/osidb.py [hunk 3]
```diff
@@ -201,6 +214,324 @@ def create_flaw(ctx, **params):
...
+        "include_fields": {"type": click.Choice(OSIDBService.get_fields(FlawComment))},
+        "exclude_fields": {"type": click.Choice(OSIDBService.get_fields(FlawComment))},
+        "include_meta_attr": {"type": click.Choice(OSIDBService.get_meta_attr_fields(FlawComment))},
+    },
+)
+@click.pass_context
+@progress_bar
+def list_flaw_comments(ctx, flaw_id, **params):
+    # TODO: handle pagination
+    # TODO: handle output
```
Comment (JimFuller-RedHat) [inline griffon/commands/entities/osidb.py:272]: all output should be handled in output.py

**Review by JimFuller-RedHat — APPROVED** [review]
lgtm - can we also bump cryptographic module as per github reported vuln ?

File: CHANGELOG.md (hunk 2 above)
Comment (JakubFrejlach) [reply CHANGELOG.md:9]: Yes it is, as there is no option to delete comments in bugzilla, once they are created, they stay there.

File: griffon/commands/entities/osidb.py (hunk 3 above)
Comment (JakubFrejlach) [reply griffon/commands/entities/osidb.py:272]: Yes, I am aware of that, I am simply putting that there so it is clear which resources needs output handling implementation.

File: griffon/commands/entities/osidb.py (hunk 1 above)
Comment (JakubFrejlach) [reply griffon/commands/entities/osidb.py:428]: Yes, that is basically just a boolean flag to tell Griffon whether editor prompt should happen. Actual editor (vi, emacs, etc.) is then obtained from the config.

**Comment by JakubFrejlach** [comment]
> LGTM - can we also bump cryptographic module as per github reported vuln ?

definitely, will add that

**Pull request merged**